registration/migrations/0092_auto_20211219_0409_squashed_0093_auto_20211219_0436.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.25 on 2021-12-20 07:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('registration', '0092_auto_20211219_0409'), ('registration', '0093_auto_20211219_0436')]
```

Comment:
Sorry, one last step to deal with this squashed migration: remove this `replaces = ...` line.  Since this hasn't been deployed anywhere else, we can treat this as a regular migration.

For further reading, see the last bullet points in [the Django docs on migration squashing](https://docs.djangoproject.com/en/2.2/topics/migrations/#migration-squashing)